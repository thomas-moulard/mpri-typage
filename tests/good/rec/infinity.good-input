val rec inf x = inf x


