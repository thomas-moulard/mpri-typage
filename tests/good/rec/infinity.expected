inf = fun x.let rec inf1 x1 =
  inf1 x1
in
inf1 x

inf : forall 'a 'b.('b -> 'a)
